README.md

```diff
@@ -37,12 +37,14 @@ Usage:
   logfmt [OPTIONS]
 
 Application Options:
-  -l, --level=       Log level filter. One of DEBUG, INFO, WARN, ERROR, FATAL (default: INFO)
-  -o, --output=      Output field selector (comma separated)
-  -e, --exclude=     Exclude field selector (comma separated)
-  -f, --filter=      Filter fields (key=value comma separated)
-  -n, --no-color     Disable color output
-  -c, --force-color  Force color output, even when outputting to a pipe
+  -l, --level=          Log level filter. One of DEBUG, INFO, WARN, ERROR, FATAL (default: INFO)
+  -o, --output=         Output field selector (comma separated)
+  -O, --output-ordered= Output field selector (comma separated) but with order by command line args, not log values
+  -e, --exclude=        Exclude field selector (comma separated)
+  -f, --filter=         Filter fields (key=value comma separated)
+  -n, --no-color        Disable color output
+  -c, --force-color     Force color output, even when outputting to a pipe
+  -s, --skip-empty      Skips outputting lines where no fields were left after output and exclude
```

Comment:
And I think it makes sense to make this the default and have an inverted flag like `--keep-empty` to still print empty lines if that's required by your setup.